docs/sources/experiments/running.md

```diff
@@ -10,92 +10,163 @@ To use a Triage experiment, you first need:
 - An experiment definition (see [Defining an Experiment](defining.md))
 
 
-## Instantiating an Experiment
 
-An `Experiment` class, once instantiated, provides access to a variety of useful pieces of information about the experiment, as well as the ability to run it and get results.
+You may run a Triage experiment two ways: through the Triage command line tool, or through instantiating an Experiment object in your own Python code and running it. The rest of this document will introduce experiment runs of increasing complexity, through both the CLI and Python interfaces.
 
-First, we'll look at how to instantiate the Experiment.
 
+## Simple Example
+
+To run an experiment, you need to tell triage at a minimum where to find the experiment file (in YAML format), how to connect to the database, and where to save trained matrices and models. In this simple example, we're assuming that the experiment will be run with only one process, and that the output should be stored on the local filesystem.
```

Comment:
This all makes sense, although the fact that you were able to make it at this particular section of the document makes me see that the paragraph is confusing: You don't actually need YAML at all to instantiate an experiment. The experiment literally just takes a dictionary and doesn't even know what YAML is. You could store it as JSON, XML, or generate it on the fly if you wanted. In fact, it's common in Triage projects to split out feature config into different YAML files and smash them together in the run script to send to the experiment. I specifically wanted to keep the experiment class itself in the dark about this.

The only reason YAML is really introduced here is for the CLI. I am open to all manner of improvements to the CLI experiment definition-reading functionality. Also, in general although we have validation routines I think the hyper-specific expectations of the experiment definition are ill-fitted for a python dictionary; synthesizing what you're suggesting here with this thought, I think we should work towards the following.

An ExperimentDefinition object, that houses the information currently held by the dictionary, but with better access (e.g. `.cohort_config.` instead of ['cohort_config']). This becomes what is passed in to the Experiment, as opposed to a basic dictionary. This object can be instantiated though the simple dictionary output of yaml.load, but could also be done through some alternate ways like you mentioned.

Reply:
Sure, I can see how that would be useful in code, both internally and in order to define your experiment in Python.

But anyway, in short, whether it applies only to the CLI or not, but particularly as we build out the CLI, some thought should likely be put into how to simplify, consolidate or otherwise organize configuration and specification of configuration.